component/main.jsonnet

```diff
@@ -16,6 +17,14 @@ local anyControllerUsesAcme = std.foldl(
   false,
 );
 
+local defaultNamespacePatch = resourcelocker.Patch(kube.Namespace('default'), {
+  metadata: {
+    labels: {
+      'network.openshift.io/policy-group': 'ingress',
```

Comment:
While debugging it, it was not obvious that the networkpolicy is depending something from the default namespace. I would prefer a specific label for the default namespace and having this explicit in the networkpolicy, so it would be clear and obvious for all the people working with.

Reply:
I'm not sure how we could do this... @srueg WDYT?

Reply:
We would have to change the label here: https://git.vshn.net/syn/commodore-defaults/-/blob/master/distribution/openshift4.yml#L44

Reply:
As discussed, I'd suggest to use `network.openshift.io/policy-group: hostNetwork` as label.